core/iwasm/libraries/libc-wasi/libc_wasi_wrapper.c

```diff
@@ -46,6 +51,8 @@ typedef struct iovec_app {
     uint32 buf_len;
 } iovec_app_t;
 
+/* If the thread manager is enabled, WASIContext is already defined */
+#if WASM_ENABLE_THREAD_MGR == 0
```

Comment:
In alternative, we can put L55-L69 in an else at L13 to make things clearer. I didn't do it to keep it in the same place as the other `typedef`s.

Reply:
could we just delete those typedefs and unconditionally include '`wasm_runtime_common.h`?

Reply:
I was referring to the two typedefs here (that are libc_wasi_wrapper specific) https://github.com/bytecodealliance/wasm-micro-runtime/blob/d2772c4153a5ca27b33df3e3d5f0aa4c804234a1/core/iwasm/libraries/libc-wasi/libc_wasi_wrapper.c#L39
Maybe I can just move them after the includes so that I can put L55-L69 in an else at L13.

Reply:
> could we just delete those typedefs and unconditionally include 'wasm_runtime_common.h

right, done